packages/web/src/components/premium-content-purchase-modal/PremiumContentPurchaseModal.tsx

```diff
@@ -98,24 +104,27 @@ const RenderForm = ({
         </Text>
       </ModalHeader>
       <ModalContent className={styles.content}>
-        <>
-          <div className={styles.contentWrapper}>
+        {stage !== PurchaseContentStage.FINISH ? (
+          <AudioMatchSection
+            amount={USDC(price / 100)
```

Comment:
I'm a little confused. In a different file, you were multiplying price by the USDC divisor (100000?) to get it into the right order of magnitude before constructing a `USDC` with it.  Isn't this going to result in a very small number when FixedDecimal pads out the value?